docs/rest/BulkDelete.http

```diff
@@ -0,0 +1,63 @@
+# .SUMMARY Sample requests for Bulk Delete
+#  __    __             __   _____     _   
+# |  \  /  \    |\  |  /  \    |      | \  |  |  |\  |
+# |  |  |  |    | \ |  |  |    |      |_/  |  |  | \ |
+# |__/  \__/    |  \|  \__/    |      | \  \__/  |  \|
+# On real data. These commands will remove all the data in your database, sometimes without a way to recover it. Use these only in a testing environment.
+
+@hostname = localhost:44348
+
+### Get the bearer token, if authentication is enabled
+# @name bearer
+POST https://{{hostname}}/connect/token
+content-type: application/x-www-form-urlencoded
+
+grant_type=client_credentials
+&client_id=globalAdminServicePrincipal
+&client_secret=globalAdminServicePrincipal
+&scope=fhir-api
+
+
+### Setup test data. Reusing the Export data since it is a good mix of resources.
+# @name batch
+POST https://{{hostname}}
+Content-Type: application/json
+Authorization: Bearer {{bearer.response.body.access_token}}
+
+< ./Data/ExportDataBatch.json
+
+### Record Bulk Delete content location
+@bulkDeleteLocation = {{bulkDelete.response.headers.Content-Location}}
+
+### Create Bulk Delete request
+# @name bulkDelete
+DELETE https://{{hostname}}/$bulk-delete
+Authorization: Bearer {{bearer.response.body.access_token}}
```

Comment:
Should these require the Prefer: async header?

Reply:
I'll add this. Do we know why the Reindex job requires the respond-sync value for Prefer? ValidateReindexRequestFilterAttribute